website/src/pages/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import Layout from '@theme/Layout'
+import styles from './index.module.scss'
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+import clsx from 'clsx'
+import IconExternalLink from '@theme/Icon/ExternalLink'
+import Link from '@docusaurus/Link'
+
+const Hero = () => {
+    const { siteConfig } = useDocusaurusContext()
+
+    return (
+        <div className={clsx('hero shadow--lw', styles.heroHeader)}>
+            <section className={styles.header}>
+                <img src={siteConfig.favicon} alt="Logo" />
+                <h1 className="hero__title">{siteConfig.title}</h1>
+            </section>
+            <section className={clsx('hero__subtitle', styles.heroSubtitle)}>
+                {siteConfig.tagline}
+            </section>
+            <section className={styles.callToActions}>
+                <Link
+                    className="button button--lg button--outline button--primary margin--lg"
+                    to="/docs/getting-started/introduction"
+                >
+                    Get Started
+                </Link>
+                <Link
+                    className="button button--lg button--outline button--link margin--lg"
+                    to="https://play.yew.rs"
+                    target="_blank"
+                >
+                    Playground
+                    <IconExternalLink />
+                </Link>
+            </section>
+        </div>
+    )
+}
+
+const FEATURES = [
+    {
+        header: 'Component Based',
+        body: 'It features a component-based framework which makes it easy to create interactive UIs. Developers who have experience with frameworks like React and Elm should feel quite at home when using Yew.',
+        to: '',
+    },
+    {
+        header: 'HTML macro',
+        body: 'Features a macro for declaring interactive HTML with Rust expressions. Developers who have experience using JSX in React should feel quite at home when using Yew.',
+        to: '',
+    },
+    {
+        header: 'HTML macro',
```

Comment:
HTML macro is listed twice. Maybe add Server Side Rendering as a third notable feature here?